### PR TITLE
[zh] Update _index.md

### PR DESCRIPTION
**Pull request by my-git9**
[zh] Update _index.md

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @tengqm

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ dockershim to other container runtimes.
...
-installations. You can find this blog post useful to understand the problem better: [Dockershim Deprecation FAQ](/blog/2020/12/02/dockershim-faq/)
+installations. Our [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq/) is there to help you
+to understand the problem better.
 -->
 自从 Kubernetes 1.20 宣布
 [弃用 dockershim](/zh/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)，
 各类疑问随之而来：这对各类工作负载和 Kubernetes 部署会产生什么影响。
-你会发现这篇博文对于更好地理解此问题非常有用：
-[弃用 Dockershim 常见问题](/zh/blog/2020/12/02/dockershim-faq/)
+我们的 [弃用  Dockershim 常见问题](/blog/2022/02/17/dockershim-faq/) 可以帮助你更好地理解这个问题。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/_index.md:29]:
```suggestion
我们的[弃用  Dockershim 常见问题](/blog/2022/02/17/dockershim-faq/)可以帮助你更好地理解这个问题。
```

**Comment by tengqm** [comment]
Please make sure you have only one commit per PR.
/lgtm
/approve

**Pull request merged**